packages/react/src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -553,7 +553,7 @@ const TimeSeriesCard = ({
           >
             <ChartComponent
               ref={(el) => {
-                chartRef = el;
+                chartRef.current = el;
```

Comment:
why do we need to pass in the ref like this instead of passing it as prop?

Reply:
That's fair. That's just the way it was, and I did't even think to change it.